azure-iot-device/setup.py

```diff
@@ -84,7 +84,6 @@
         # Actual project dependencies
         "six>=1.12.0,<2.0.0",
         "paho-mqtt>=1.4.0,<2.0.0",
-        "transitions>=0.6.8,<1.0.0",
```

Comment:
If we are no longer dependent on this can you please remove it from the ThirdPartyNotice.txt

Reply:
+1

---
In reply to: [440493584](https://github.com/Azure/azure-iot-sdk-python/pull/592#discussion_r440493584) [](ancestors = 440493584)